chef/cookbooks/ceph/resources/client.rb

```diff
@@ -1,6 +1,13 @@
 actions :add
 default_action :add
 
+# Whether we have ceph cluster deployed with non standard configuration
+# and ceph.conf and admin keyring is placed somewhere else than in /etc/ceph
+# This can happened when we want to use external ceph cluster deployed 
+# without crowbar
```

Comment:
I am not able to parse this :)

Do you mean something like:
"Allow using non-standard locations for ceph.conf and admin keyring. This can happen when using and external ceph cluster not deployed with crowbar"

Two other things I don't understand:
1. Why would an external ceph cluster result in different config file locations on the client?
2. Where does the client actually get the ceph.conf and admin.keyring from?